### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Nahid1911**
Update the webpage for desktop users, with all the required fields.

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -7,33 +7,44 @@
...
+         
         </ul>
-
-        <a class="disable-button"><img src="media/Disabled.svg" alt=""></a>
+        <div class="disable-img">
+          <a class="disable-button"><img src="media/Disabled.svg" alt=""></a>
+        </div>
  
       </div>
       <section class="work-container">
```
Comment (leonardodiasb) [inline Index.html:50]: - [ ] Kindly add some margin at the top to separate the first section from the projects section like the Figma template.
 - Your design:
 - 
image

 - Figma template:
 - 
image

File: style.css [hunk 2]
```diff
@@ -358,3 +388,432 @@
...
+    border-radius: 5px;
+  }
+
+  .project-info {
+    display: flex;
+    flex-direction: column;
+    border-radius: 8px;
+    text-align: center;
+    justify-content: center;
+  }
```
Comment (leonardodiasb) [inline style.css:552]: - [ ] Kindly centralize all the elements in the card and add some padding/margin to the button so it does not stay on the edge of the project card.
 - Your design:
 - 
image

 - Figma template:
 - 
image

File: Index.html [hunk 3]
```diff
@@ -231,13 +242,17 @@ <h2>Skills</h2>
...
+          </p>
         </div>
         <div class="input-elements">
           <form action="https://formspree.io/f/meqwrjvl" method="post">
-            <input type="text" id="fname" name="fname" placeholder="Full Name" maxlength="30" required><br>
+            <input type="text" class="full-name" id="fname" name="fname" placeholder="Full Name" maxlength="30" required><br>
+            <div class="first-last-name">
+              <input type="text" class="first-name" id="fname" name="fname" placeholder="First Name" maxlength="30" required><br>
+              <input type="text" class="last-name" id="fname" name="fname" placeholder="Last Name" maxlength="30" required>
+            </div><br>
```
Comment (leonardodiasb) [inline Index.html:255]: - [ ] As the project required, templates that has 2 input fields on the desktop version should be ignored and remain with one input field for 'full name'. Kindly change the inputs to make the form consistent with only the 'full name' input field for both mobile and desktop versions.
 - [ ] Also, please make sure that the input field size is consistent and does not differ from the other input field sizes, like the email field below.
 - 
image

File: Index.html (hunk 1 above)
Comment (Nahid1911) [reply Index.html:50]: resolved

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @Nahid1911 , 



Great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**